src/test/scala/chiselTests/aop/SelectSpec.scala

```diff
@@ -228,4 +229,68 @@ class SelectSpec extends ChiselFlatSpec {
     intercept[Exception] { Select.instances(top) }
   }
 
+  "Select currentInstancesIn and allCurrentInstancesIn" should "support module, extmodule, and D/I" in {
+    import chisel3.experimental.hierarchy._
+
+    class MyLeafExtModule extends ExtModule {}
+
+    class MyLeafModule extends RawModule {}
+
+    class MyIntermediateModule extends RawModule {
+      Module(new MyLeafModule)
+      Module(new MyLeafModule)
+    }
+
+    @instantiable
+    class MyLeafInstance extends RawModule {}
+
+    @instantiable
+    class MyIntermediateInstance extends RawModule {
+      val definition = Definition(new MyLeafInstance)
+      Instance(definition)
+      Instance(definition)
+    }
+
+    class MyWrapperModule extends RawModule {
+      val definition = Definition(new MyIntermediateInstance)
+      Module(new MyIntermediateModule)
+      Instance(definition)
+      Module(new MyLeafExtModule)
+
+      atModuleBodyEnd {
+        implicit val mg = new chisel3.internal.MacroGenerated {}
+
+        val myInstances = Select.unsafe.currentInstancesIn(this).map(_._lookup { m => m.name })
+
+        (myInstances should contain).theSameElementsInOrderAs(
+          Seq("MyIntermediateModule", "MyIntermediateInstance", "MyLeafExtModule")
+        )
```

Comment:
```suggestion
        myInstances should be(Seq("MyIntermediateModule", "MyIntermediateInstance", "MyLeafExtModule"))
```
Any reason we can't use a simpler equality check?

Reply:
uh no, probably not. i was just searching in the scala test docs. `be(Seq(...))` sounds good to me.

Reply:
Same comment below.